homeassistant/components/tuya/switch.py

```diff
@@ -697,6 +697,15 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Smart Water Timer
+    # UNDOCUMENTED
+    "sfkzq": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
+            name="Water Valve (Set Countdown Required)",
```

Comment:
Entity names are not made for instructions.

Reply:
Understood. I'll make the change and re-commit.